### DISCUSSION TITLE
How to style GroupBox without loosing MaterialDesignInXamlToolkit features, to have rounded corners?

### DISCUSSION THREAD

**Discussion by sensboston** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3455] [Q&A]
How to style GroupBox without loosing MaterialDesignInXamlToolkit features, to have rounded corners?

**Comment by nicolaihenriksen** [comment]
@sensboston Currently there is not a great way of extending the default `GroupBox` style to use rounded corners apart from making a complete local copy of the style (which is of course doable).

You could consider using the `MaterialDesignCardGroupBox` style to get rounded corners, but you still cannot control the `CornerRadius` without making a complete local copy of the style.

```xaml
<GroupBox Header="Card Look" Style="{StaticResource MaterialDesignCardGroupBox}">
  <TextBlock Text="My Content" />
</GroupBox>
```

![image](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/assets/19572699/edec489a-5dc0-4209-9705-24028e8f8661)

**Comment by sensboston** [comment]
Thank you! It's enough for now; maybe devs will add that property in future.
